charts/m4d/values.yaml

```diff
@@ -188,4 +188,57 @@ clusterScoped: true
 
 # Set to true to enable socat in the manager pod to forward
 # traffic from a localhost registry. Used only for development.
-integrationTests: false
\ No newline at end of file
+integrationTests: false
+
+# Values set in m4d-config ConfigMap.
+config:
+  # Vault related values set in m4d-config ConfigMap.
+  vault:
+    # Set to the Vault address.
+    address: "http://vault.m4d-system:8200/"
+    # Set to the path where credentials of dataset accessed by the m4d are stored.
+    datasetHome: "m4d/dataset-creds/"
+    # Set to the dataset credentials secret engine mount path.
+    datasetMount: "v1/sys/mounts/m4d/dataset-creds"
+    # Set to the path where user credentials are stored.
+    userHome: "m4d/user-creds/"
+    # Set to the user credentials secret engine mount path.
+    userMount: "v1/sys/mounts/m4d/user-creds"
+    # Set to the amount of time the authorization issued by Vault is valid.
+    ttl: "24h"
+    # Set to the Vault address that stores the credentials to data source for data assets.
+    userVaultAddress: "http://vault:8200/"
+    # Set to the path where credentials to data source for data assets are stored.
+    userVaultPath: "external"
+    # Set to the name of the role that modules use to access dataset credentials.
+    modulesRole: "module"
+
+  catalog:
+    # Set to the catalog URL.
+    url: "https://lab-core.egeria-catalog:9443"
```

Comment:
`url` is specific configuration of the egeria connector, not of "catalog".  We also don't deploy egeria connector as part of the chart so it shouldn't be default.

Reply:
@roee88 I don't think it should be here at all - service url, policy name, ... Instead, the connectors should use their own configmaps.

Reply:
I agree, but that's what we currently have and this PR is focused on the chart. I will need to iterate on that in follow-up PRs.